README.md

```diff
@@ -44,6 +44,18 @@ This is a fork of [inhouse_bot](https://github.com/mrtolkien/inhouse_bot). The o
 
  7. ```docker compose up -d```, at this point the application should be running
 
+# How Algorithm Works
+1. Generate all possible teams with players in queue
+2. Get each players summoner rift rank (Solo/Duo)

```

Comment:
Is it possible to get the highest value between their Solo, and Flex rank here? I've seen people in diamond flex and gold Solo because they don't play it.

Reply:
The [flex rank is accounted for](https://github.com/adamschachne/inhouse_bot/blob/72-refactoring-matchmaking/inhouse_bot/common_utils/lol_api/tasks.py#L33) but it prefers the solo rank. 

I agree that it should probably be the greater of the two, and this readme should say Solo/Duo and Flex

Reply:
This is a real tricky case, I feel like flex is not a real rank to quantify skill of a player, since in flex you are playing with a crew/friends who know you and your style, but in an inhouse environment you have to use your solo ability and then start to use your game knowledge with the current players on your team

Reply:
Basically, I feel like there are way more false positives with someone being high elo in flex compared to high in solo duo.